modules/ingestor/src/endpoints/advisory.rs

```diff
@@ -37,27 +37,6 @@ pub async fn upload_advisory(
     Ok(HttpResponse::Created().json(advisory_id))
 }
 
-#[utoipa::path(
```

Comment:
I think it's ok to remove it from here, but it should end up somewhere else. I can't find that, did I overlook it?

Reply:
It's [already in the fetch module](https://github.com/trustification/trustify/blob/bc2c73fde8fd0b433c9facd784afb556846f5dee/modules/fetch/src/endpoints/advisory.rs#L37-L51), so not a part of this PR

Reply:
Yea, there's a version with the digest, which I am not sure we use much. I think it makes sense having both versions with the `id`.

Reply:
I'll leave these in for now, but we need to agree on what `xxx` is in this request: `/api/v1/advisory/{xxx}`. Is it the entity's `id` field or its `sha256` field? Or do we need an api for both?

Complicating matters further, these will also work:

- `/api/v1/advisory?q=id%3Dfoo`
- `/api/v1/advisory?q=sha256%3Dfoo`

Reply:
I would go for the `id`.

Reply:
But we'd prefer to not leak IDs. And sha can he deterministic.

Reply:
Yeah, I thought we agreed IDs wouldn't be a part of the public API.

Potentially relevant: #157 #158